docs/UserGuide.md

```diff
@@ -935,65 +953,38 @@ The `sort` command is designed to help you **quickly and conveniently organize**
 
 With this command, you can prioritize and manage your applications according to a variety of criteria.
 
-#### Command Usage
+##### Command Usage
 
 **Command**:
 ```
-sort [n/COMPANY_NAME] [r/ROLE] [c/CYCLE] [s/STATUS] [d/DEADLINE]
+sort [n/ORDER] [r/ORDER] [c/ORDER] [s/ORDER] [d/ORDER]
 ```
 
-#### Parameters:
+##### Parameters:
 
 <box type="important" seamless>
 
-* At least **one** of the optional parameters must be provided.
-* Each parameter must be replaced by either `a` (for ascending order) or `d` (for descending order). 
+* Exactly **one** of the optional parameters must be provided.
+* The `ORDER` parameter must only be replaced by either `a` (for ascending order) or `d` (for descending order). 
 
 </box>
 
-* **n/COMPANY_NAME** (Optional): Sorts applications by the company name.
-* **r/ROLE** (Optional): Sorts applications by the role.
-* **c/CYCLE** (Optional): Sorts applications by the cycle.
-* **s/STATUS** (Optional): Sorts applications by their status.
-* **d/DEADLINE** (Optional): Sorts applications by their deadline.
-
-#### Example:
+* **n/ORDER** (Optional): Sorts applications by the company name.
+* **r/ORDER** (Optional): Sorts applications by the role.
+* **c/ORDER** (Optional): Sorts applications by the cycle.
+* **s/ORDER** (Optional): Sorts applications by their status.
+* **d/ORDER** (Optional): Sorts applications by their deadline.
 
-1. **Sort by Company Name (Ascending)**
-   ```
-   sort n/a
-   ```
+##### Example:
```

Comment:
#145 resolved here! Feel free to add more examples to help with explaining stuff like sorting of deadlines. I think that will not be excessive